docs/CONFIGURATION-EXAMPLES.md

```diff
@@ -0,0 +1,142 @@
+<!---
+id: configuration
+title: Configuring the Testrunner Toolkit
+sidebar_label: Configuration
+--->
+# Configuring the Testrunner Toolkit
+The Testrunner Toolkit (`saucectl`) requires a configuration file to know which tests to run along with which framework to use.
+
+## Basic Configuration
+
+By default, `config.yml` is the file `saucectl` looks to for its configuration.
+
+__Simple `config.yml` using `puppeteer`__
+
+```yaml
+apiVersion: v1
+metadata:
+  name: Feature XYZ
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests/**/*.js
+image:
+  base: saucelabs/sauce-puppeteer-jest-node
+  version: v0.1.0
+sauce:
+  region: us-west-1
+```
+
+If you wish to use more than one framework, or to configure different sets of tests separately, you can use any name for the configuration file and specify it with the following command:
+
+```bash
+saucectl run -c ./path/to/config.yml
+```
+
+## Configuration Examples
+Below are framework-specific configuration examples that exist in the [Testrunner Toolkit repository](https://github.com/saucelabs/testrunner-toolkit/tree/master/.sauce). The repository uses these configurations for its pipeline:
+
+>
+> **NOTE:** The test files need to match `(spec|test)` in their file name so they are automatically detected as `testfiles`.
+>
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Puppeteer-->
+<br />
```

Comment:
We aren't using Docusaurus for this project, this would need to be added first.

Reply:
Yep, forgot to remove that those tags, I have that example in a separate branch here: https://github.com/spider-sauce/testrunner-toolkit/tree/docusaurus-poc. 

Good catch.